src/embedding/embedding.jl

```diff
@@ -127,6 +153,28 @@ function generator_minimum_polynomial(f::FinFieldMorphism)
 
     return berlekamp_massey(A, d)
 end
+=#
+
+function generator_minimal_polynomial(f::FinFieldMorphism)
+
```

Comment:
There are quite a few blank lines after prototypes here. I realise they were there in the original, but perhaps they can be removed, as we don't do that anywhere else.